### PR TITLE
Remove unnecessary `this.get()` calls

### PR DESCRIPTION
**Pull request by Turbo87**
With the newer Ember.js versions this is no longer necessary in the majority of cases 🎉 

r? @locks

### CODE REVIEW

File: app/controllers/crate/version.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ export default Controller.extend({
     return this.get('crate.owner_user').findBy('id', this.get('session.currentUser.id'));
   }),
   notYankedOrIsOwner: computed('model', 'crate.owner_user', 'session.currentUser.id', function () {
-    return !this.get('model').yanked || this.get('crate.owner_user').findBy('id', this.get('session.currentUser.id'));
+    return !this.model.yanked || this.get('crate.owner_user').findBy('id', this.get('session.currentUser.id'));
```
Comment (locks) [inline app/controllers/crate/version.js]: I tried tracing the logic and I think `this.model` comes from Ember Data so we might want to leave the `get` here.

File: app/controllers/me/index.js [hunk 2]
```diff
@@ -22,7 +22,7 @@ export default Controller.extend({
   emailNotificationsSuccess: false,
 
   setAllEmailNotifications(value) {
-    this.get('ownedCrates').forEach(c => {
+    this.ownedCrates.forEach(c => {
```
Comment (locks) [inline app/controllers/me/index.js]: `ownedCrates` comes from `ownedCrates: alias('model.ownedCrates')`, no get necessary in this file?

File: app/controllers/crate/version.js (hunk 1 above)
Comment (Turbo87) [reply app/controllers/crate/version.js]: removed

File: app/controllers/me/index.js (hunk 2 above)
Comment (Turbo87) [reply app/controllers/me/index.js]: removed for now

**Pull request merged**